### DISCUSSION TITLE
Enhance mobile UI

### DISCUSSION THREAD

**Discussion by isolume** [yhype/yhype-feedback#6] [Ideas]
Currently, the UI on mobile is not enjoyable to say the least, and to view statistics you are nearly forced to enable desktop mode on the browser you are using. I suggest fixing/improving the mobile UI.

**Comment by antonkomarev** [comment]
Thank you for the feedback @n3rd3x3! Could you share a screenshot of what you want to improve?

**Reply by r2hu1** [reply]
why the source code is private? make it public and open for contributions

**Reply by antonkomarev** [reply]
I'm not ready for it at this moment. But I surely will think about it later.

**Reply by antonkomarev** [reply]
@isolume @r2hu1 
You can try new interface, it includes dark theme and has mobile support now!

<img width="1511" height="829" alt="image" src="https://github.com/user-attachments/assets/22065a8d-688e-4184-b03e-00bfb2766bec" />
